gocd/templates/relay.jsonnet

```diff
@@ -12,6 +12,10 @@ local pipedream_config = {
       destination: 'relay',
     },
   },
+  rollback: {
+    material_name: 'relay_repo',
+    stage: 'deploy-production',
+  },
```

Comment:
Only this this like adding new rollback generated pipeline.

Reply:
Yep, these values are from the `gocd/templates/libs/relay.libsonnet` file, the `relay_repo` is the material name used in each repo for github.com/getsentry/relay, which the rollback pipeline used to lookup the SHA we want to rollback to and the `stage` is the name of the stage the rollback pipeline will re-run during a rollback.

I'm putting together some docs on pipedream and rollbacks here: https://www.notion.so/sentry/Pipedreams-in-GoCD-with-Jsonnet-430f46b87fa14650a80adf6708b088d9?pvs=4#0fdf3335c8154303ab9dda719f44c697

Not complete, but I'm intending to link to the docs in the relay.jsonnet file when switching over to pipedream next week.